arch/aarch64/mcount-support.c

```diff
@@ -135,7 +136,14 @@ void mcount_get_stack_arg(struct mcount_arg_context *ctx,
 	if (offset < 1 || offset > 100)
 		pr_dbg("invalid stack offset: %d\n", offset);
```

Comment:
Oops, sorry, you should handle this case as well.  It'd be nice call `memset()` with 0 and return to prevent invalid values.

Reply:
If a return is made in this code, it is not necessary to check the memory range.
If the offset is between 1 and 100, it does not exceed the memory range.
Why is the code limit the offset range between 1 and 100?

Reply:
Because it's very unlikely to have arguments more than 100.

Reply:
I added code to handle this case.